lxd/instance_console.go

```diff
@@ -505,8 +506,8 @@ func instanceConsolePost(d *Daemon, r *http.Request) response.Response {
 	ws.height = post.Height
 	ws.protocol = post.Type
 
-	resources := map[string][]string{}
-	resources["instances"] = []string{ws.instance.Name()}
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", ws.instance.Name(), "console")}
```

Comment:
Lets just use the instance URL without the console part, as that isnt its own resource.